Airports/EGCC/Fixes.txt

```diff
@@ -0,0 +1,3 @@
+DOMIG N053.32.11.720 W001.53.03.470
```

Comment:
I'd add in 

C23LI : 532905.02N 0015934.99W
C23LF : 532712.82N 0020329.44W

as well 😃

Reply:
Done